swagger.yaml

```diff
@@ -1112,6 +1112,152 @@ paths:
             application/json:
               schema:
                 $ref: "#/components/schemas/Error"
+
+  /v1/projects/{projectId}/notifications/{notificationId}:
+    delete:
+      operationId: deleteNotification
+      summary: Delete a notification
+      description: Delete a notification, only for testing purposes.
+      tags:
+        - Notifications
+      parameters:
+        - name: projectId
+          in: path
+          required: true
```

Comment:
Notification object itself doesn't have a link to a Project, why are we using a projectId as a parameter? Should it be a taskId? Or maybe even just notificationId is enough?

Reply:
It's for testing purposes only so you are right, just notificationId is good enough. This was leftover from when they were kept under "project"